jobs/ci-kubernetes-e2e-gce-federation-serial.env

```diff
@@ -1,5 +1,4 @@
 ### job-env
-FEDERATION=true
```

Comment:
Only cleaned up `FEDERATION_UP`, `FEDERATION_DOWN`, and `JENKINS_FEDERATION_PREFIX`